### PR TITLE
Where is julia path for windows.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by StefanKarpinski — APPROVED** [review]
LGTM assuming this is correct which an actual Windows user may want to verify.

File: README.windows.md [hunk 1]
```diff
@@ -58,8 +58,9 @@ The 64-bit (x86_64) binary will only run on 64-bit Windows and will otherwise re
 
  1. [Download](https://julialang.org/downloads) the latest version of Julia.
     Extract the binary to a reasonable destination folder, e.g. `C:\julia`. By default, 
-    Julia will installed into `C:\Users\YOURNAME\AppData\Local\Julia-1.0.0\bin`, You can put this path
-    to windows PATH env variable. YOURNAME was your windows user name.
+    Julia will be installed into `C:\Users\YOURNAME\AppData\Local\Julia-1.0.0\bin`
+    (YOURNAME being your Windows user name). You can put this path
```
Comment (KristofferC) [inline README.windows.md]: Put quotes around "YOURNAME" (`YOURNAME`).

**Comment by netroby** [comment]
Thanks for your help.

**Comment by bramtayl** [comment]
Probably want to change put to add

**Comment by netroby** [comment]
@bramtayl  Good suggest. changed

**Review by StefanKarpinski — APPROVED** [review]

**Comment by thchr** [comment]
If https://github.com/JuliaLang/julia/issues/29106 is addressed at some point, it should simultaneously adjust/update the `README.windows.md` changes introduced here as well. Just posting to add some breadcrumbs.

**Comment by tkelman** [comment]
Adding julia's bin folder permanently to your path is an extremely bad idea as that's where every dependency dll is located. This will break other software that happens to depend on openblas, gmp, etc but requires incompatible versions.

**Comment by nalimilan** [comment]
Good point. I've removed the mention of PATH, so that we can go ahead and merge this PR.

**Comment by w1res** [comment]
Is there any plan/desire to come up with some workaround so that people on Windows can use julia on path?

Additionally, it might make sense to add information to the README specifically stating _not_ to add it to path on Windows. I had already added it to path and didn't know it was a bad idea until someone mentioned this thread form a [discourse post](https://discourse.julialang.org/t/what-is-the-best-way-to-set-the-path-to-julia-on-windows-10/16467). Granted that this is in the beginner forum, it seems that other people on Windows _are_ in fact adding julia to path.

**Pull request closed without merging**